c/tskit/tables.h

```diff
@@ -961,19 +961,61 @@ on and may change arbitrarily between versions.
 */
 void tsk_individual_table_print_state(const tsk_individual_table_t *self, FILE *out);
 
-/** @} */
+/**
+@brief Replace this table's data by copying from a set of column arrays
 
-/* Undocumented methods */
+@rst
+Clears the data columns of this table and then copies column data from the specified
+set of arrays. The supplied arrays should all contain data on the same number of rows.
+The metadata schema is not affected.
+@endrst
 
+@param self A pointer to a tsk_individual_table_t object.
+@param num_rows The number of rows to copy from the specifed arrays
+@param flags A pointer to the tsk_flags_t array of flags to be copied
+@param location A pointer to the double array of locations to be copied
+@param location_offset A pointer to the tsk_size_t array of location offsets to be copied
+@param parents A pointer to the tsk_id_t array of parents to be copied
+@param parents_offset A pointer to the tsk_size_t array of parent offsets to be copied
+@param metadata A pointer to the char array of metadata to be copied
+@param metadata_offset A pointer to the tsk_size_t array of metadata offsets to be copied
+@return Return 0 on success or a negative value on failure.
```

Comment:
Nitpicking *even more*, but some of these have periods at the end, and others not. I vote to add periods. =)